mypy/types.py

```diff
@@ -103,6 +103,8 @@ class TypeOfAny:
     from_another_any = 7  # type: Final
     # Does this Any come from an implementation limitation/bug?
     implementation_artifact = 8  # type: Final
+    # Does this Any come from use in the suggestion engine?
```

Comment:
Maybe also explain that this allows these `Any` types to be ignored when inferring suggestion, resulting in more precise types.